rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -0,0 +1,88 @@
+- project:
+    name: 'RPC-Artifact_Build-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - artifacts:
+          branch: artifacts-14.0
+          branches: "artifacts-.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
+      - python:
+          ARTIFACT_BUILD_SCRIPT: "scripts/artifacts-building/python/build-python-artifacts.sh"
+      - containers:
+          ARTIFACT_BUILD_SCRIPT: "scripts/artifacts-building/containers/build-process.sh"
+    ztrigger:
+      - pr:
+          CRON: ""
+      - periodic:
+          branches: "do_not_build_on_pr"
+          PUSH_TO_MIRROR: yes
+    jobs:
+      - 'RPC-Artifact_Build-{series}-{image}-{context}-{ztrigger}'
+
+- job-template:
+    # DEFAULTS
+    IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    CRON: "H H(9-21) * * 1-5"
+    STAGES: "Allocate Resources, Connect Slave, Build Artifacts, Cleanup"
+    branch: artifacts-14.0
+    ARTIFACT_BUILD_SCRIPT: "/scripts/bootstrap-aio.sh"
+
+    # TEMPLATE
+    name: 'RPC-Artifact_Build-{series}-{image}-{context}-{ztrigger}'
+    project-type: workflow
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      # See params.yml
+      - rpc_params:
+         RPC_BRANCH: "{branch}"
+      - rpc_gating_params
+      - single_use_slave_params:
+         IMAGE: "{IMAGE}"
+         FLAVOR: "performance2-15"
+      - string:
+          name: PUSH_TO_MIRROR
+          default: "NO"
```

Comment:
Probably should be ```{PUSH_TO_MIRROR}``` That variable also needs to be added to defaults at the top of job-template.